dashboard/src/components/DocsSection.tsx

```
@@ -0,0 +1,311 @@
+
+      {/* External Links */}
+      <div
+        className="card-postit text-center"
+        style={{ transform: 'rotate(-0.5deg)' }}
+      >
+        <h4 className="text-xl font-bold text-[#2d2d2d] mb-4">More Resources</h4>
+        <div className="flex flex-wrap justify-center gap-4">
+          <a
+            href="https://docs.manim.community/"
+            target="_blank"
+            className="flex items-center gap-2 px-4 py-2 bg-white border-2 border-[#2d2d2d] shadow-hard-sm hover:shadow-none hover:translate-x-[2px] hover:translate-y-[2px] transition-all"
```

Biome (2.3.13)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)